### ISSUE TITLE
EDD 3.0 - adjust styling, label on discount amount

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#7713]
### Explain your enhancement (please be detailed)
I'm looking at the updated discounts page in EDD 3.0 and have encountered two issues:
* if I change the discount type to `%`, the `%` character is cut off visually
* the discount type has no accessible text or label. A screen reader user will only hear "percent sign" and "dollar sign" without any indication of what those are changing

Visually, my suggestion would be to replace the current CSS, which uses a lot of absolute positioning and thus requires padding to shim things out, which is not flexible, with something like flexbox, which could give the a very similar visual appearance but with more adaptability for different characters.

Not having a label on the select field is problematic for accessibility, but I understand the desire to visually unify the two inputs. The select field has no label. One option would be to add a visually hidden label, such as:

```html
<label class="screen-reader-text" for="edd-amount-type">Amount Type</label>
```
which would keep the visual appearance as it is, but provide context for screen reader users. It might also be helpful, if possible, to use words for screen reader users instead of the symbols in the select element.

I know historically, the amount type has come before the discount amount in the settings page, but with this visual approach, it might be worth putting the type after the amount, as I'm reading this in my mind as a discount of "ten dollars" or "ten percent".

### Justification or use case
When we're doing creative things visually, we need to make sure they work regardless of screen/browser zoom/selection, and absolute positioning doesn't really lend itself to that. And we do not want to reduce the accessibility of this feature with the new major version, please.

**Comment by robincornett** [comment]
![image](https://user-images.githubusercontent.com/4853066/78609162-2273d000-7830-11ea-9fa6-6563cd24c2a6.png)
Windows 10, Firefox 74.0.1 (64 bit), zoom level does not seem to matter

It isn't cut off by a huge amount, but I'm thinking if the `$` is translated to a wider currency symbol, since it's a translatable string, then that also could be cut off (or if someone decides that they want to add a custom translation and change that to a word like "Flat" and `%` to "Percentage" since users can upload custom translation files).

**Comment by robincornett** [comment]
Hypothetically, with a change to flexbox, you could end up with it looking like this:
![image](https://user-images.githubusercontent.com/4853066/78673974-e46ac080-78b0-11ea-94cb-5f03d44f5f80.png)
You can remove the `edd-amount-type-select-wrapper` span. There is still a bit of border style tweaking to make it appear as one unit, but it's simpler CSS, and more flexible with the translations.

**Comment by spencerfinnell** [comment]
@robincornett I'd definitely like to see a flexbox solution here. I do think there should be a division between the type selector and amount input though. 

> I know historically, the amount type has come before the discount amount in the settings page, but with this visual approach, it might be worth putting the type after the amount, as I'm reading this in my mind as a discount of "ten dollars" or "ten percent".

I think we should explore this too. `%` _always_ makes sense after, and the currency symbol makes sense after for some currencies.
